examples/features/grid-paging.js

```diff
@@ -0,0 +1,119 @@
+Ext.require([
+    'Ext.container.Container',
+    'Ext.panel.Panel',
+    'Ext.grid.Panel',
+    'GeoExt.component.Map',
+    'GeoExt.data.store.WfsFeatures'
+]);
+
+Ext.Loader.loadScript({
+    url: '../../classic/toolbar/WfsPaging.js'
+});
+
+var olMap;
+var gridWest;
+var featStore;
+
+Ext.application({
+    name: 'FeatureGridWithPaging',
+    launch: function() {
+        // a WFS feature store
+        featStore = Ext.create('GeoExt.data.store.WfsFeatures', {
+            model: 'GeoExt.data.model.Feature',
+            passThroughFilter: true,
+            createLayer: true,
+            url: 'https://maps.dwd.de/geoserver/dwd/ows?',
```

Comment:
An awesome service! According to the `AccessConstraints` of the [capabilities](https://maps.dwd.de/geoserver/ows?service=wfs&version=2.0.0&request=GetCapabilities) which link to the [Copyright](https://www.dwd.de/DE/service/copyright/copyright_node.html) (German, [English version](https://www.dwd.de/EN/service/copyright/copyright_artikel.html)) we have to attribute correctly (if you follow another link you'll end up [here](https://www.dwd.de/EN/service/copyright/templates_dwd_as_source.html)) like this: `Source: Deutscher Wetterdienst`.

I am unsure whether this is an easy change, but I think we should enable it. Probably in follow-up PR, so this doesn't get to big.